### DISCUSSION TITLE
Call to undefined method Deployer\Host\Host::hostname() in beta 7.0

### DISCUSSION THREAD

**Discussion by AlexandreKilian** [deployphp/deployer#2287] [Help needed]
- Deployer version: 7.0.0-beta5
- Deployment OS: macOs

when using the option `hostname`, following exception is thrown: `Call to undefined method Deployer\Host\Host::hostname()``
Must have gotten lost in the update, couldnt find any indices it had been renamed either.

**Comment by antonmedv** [comment]
Please read https://github.com/deployphp/deployer/blob/master/docs/UPGRADE.md#upgrade-from-6x-to-7x

**Comment by AlexandreKilian** [comment]
why is the config file beeing renamed with every version? 😅

**Reply by antonmedv** [reply]
It's not 🙃

**Comment by antonmedv** [comment]
When hosts() has setHostname() before?)

**Comment by AlexandreKilian** [comment]
According to the upgrade guide:
> Change real_hostname to hostname.

But hostname isn't defined 😅

**Reply by antonmedv** [reply]
This is correct, the configuration was renamed. As well as the setter function.

**Comment by AlexandreKilian** [comment]
Also  `Call to undefined method Deployer\Host\Host::alias()`

**Reply by antonmedv** [reply]
Read https://github.com/deployphp/deployer/blob/master/docs/UPGRADE.md#upgrade-from-6x-to-7x one more time.

**Comment by antonmedv** [comment]
Also, consider donating https://github.com/sponsors/antonmedv 😉
A few bucks will be cool.
